src/app.ts

```diff
@@ -113,28 +113,31 @@ async function app() {
     return;
   }
 
-  // Ensure FHIR R4 is added as a dependency
-  const fhirR4Dependency = packageJSON.dependencies?.['hl7.fhir.r4.core'];
-  if (fhirR4Dependency !== '4.0.1') {
+  // Ensure FHIR R4 is added as a fhirVersion
+  const dependencies = config.dependencies;
```

Comment:
`config.dependencies` will be `undefined` if there were no dependencies listed in the configuration file, so an empty list should be used as a backup.

Reply:
Thanks for catching that, fixed in https://github.com/FHIR/sushi/pull/418/commits/751800e0d5398398eff0f617aab1bcc8c1a451d3